py/jpy/src/main/c/jni/org_jpy_PyLib.c

```diff
@@ -2145,6 +2145,21 @@ JNIEXPORT jboolean JNICALL Java_org_jpy_PyLib_hasGil
     return result;
 }
 
+/*
+ * Class:     org_jpy_PyLib
+ * Method:    ensureGil
+ * Signature: (Ljava/util/function/Supplier;)Ljava/lang/Object;
+ */
+JNIEXPORT jobject JNICALL Java_org_jpy_PyLib_ensureGil
```

Comment:
Add to `.h` file as well.

Reply:
Is there a reason we don't package the generated .h files that the build emits?

Reply:
I don't know.